centreon/packages/ui/package.json

```diff
@@ -37,13 +43,19 @@
     "@storybook/manager-webpack5": "^6.5.11",
     "@storybook/mdx2-csf": "^0.0.3",
     "@storybook/react": "^6.5.11",
+    "@testing-library/cypress": "^8.0.7",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/react-hooks": "^8.0.1",
     "@types/jest": "^27.5.0",
     "@types/ramda": "^0.28.12",
     "axios-mock-adapter": "^1.20.0",
     "babel-plugin-require-context-hook": "^1.0.0",
     "clean-webpack-plugin": "^4.0.0",
+    "cypress": "^12.8.0",
     "identity-obj-proxy": "^3.0.0",
+    "mochawesome": "^7.1.3",
     "puppeteer": "^13.7.0",
+    "react": "^18.2.0",
```

Comment:
should be a peer dependency ?

Reply:
No, Cypress need react for run his tests.